reference/authorisation_model.md

```diff
@@ -0,0 +1,132 @@
+# JAAS: Authorisation Model
+
+JAAS uses a Relationship-Based Access Control (ReBAC) scheme for authorisation purposes. This document illustrates the underlying authorisation model used by JAAS.
+
+```{tip}
+For an explanation on Relationship-Based Access Control (ReBAC) check out :doc:`this <../explanation/authorisation>` explanation topic.
```

Comment:
In-tree documentation references are not correct for markdown, I suggest looking for all `:doc:` and fixing these.

Reply:
Fixed, thanks!